kDriveCore/Data/Cache/DriveFileManager.swift

```diff
@@ -140,6 +140,17 @@ public final class DriveFileManager {
                     }
                 }
             }
+
+            // Migration for APIV3
+            if oldSchemaVersion < 18 {
+                migration.enumerateObjects(ofType: UploadFile.className()) { oldObject, newObject in
+                    guard let newObject else {
+                        return
+                    }
+
+                    newObject["uploadingSession"] = nil
```

Comment:
Nilifying existing session, forcing to re-create one. May not be perfect but I did not have issues doing a migration this way.